### ISSUE TITLE
facebook-config-ui instructions are outdated

### ISSUE THREAD

**Issue by stevenhao** [meteor/meteor#8867]
Due to some recent changes in Facebook's developer portal, the instructions provided by accounts-ui are slightly unclear.

Some background: https://guide.meteor.com/accounts.html#accounts-ui suggests using the package `accounts-ui` with a package like `accounts-facebook` as a quick way to prototype an accounts system for your app.  The `facebook-config-ui` package presents a modal for the developer to enter the App ID and App Secret, as well as a guide through the process of creating and configuring a facebook app to support Facebook Login.

It seems like facebook recently updated their developer portal, and our guide is no longer accurate. In fact, [facebook's very own guide](https://developers.facebook.com/docs/apps/register) seems to be out of date as well. 

Some key differences:
- Right after creating a new Facebook App, you are asked to give it a name, instead of being shown a "Choose a Platform" page
- After giving it a name, you are redirected to a "Select a Product" page
- Only after selecting the "Facebook Login" product do you get to choose a platform

[I'm not sure if this counts as a bug -- sorry if this issue actually belongs somewhere else]

**Comment by abernix** [comment]
Definitely should be fixed!  The link you posted is for the Guide, but the text in the screenshots is from the Meteor codebase ([here, I think](https://github.com/meteor/meteor/blob/devel/packages/facebook-config-ui/facebook_configure.html#L22)) so this is the right place to report it if it's still wrong in the core.  

On the other hand, if it's just the screenshots that need updating or some text on the Guide, then it should be reported/PR'd to github.com/meteor/guide/.  Want to submit a PR so we can improve the UX either way?

**Comment by hwillson** [comment]
I've confirmed this needs to be updated in both core and in the Guide. Marking as `pull-requests-encouraged` (and when the changes in core are in place, I'll create a new issue in the Guide repo).

**Comment by hwillson** [comment]
PR #9258 should help with the out of date `facebook-config-ui` instructions. After those changes have been vetted and hopefully merged, I'll then take care of updating the Guide accordingly.
